components/cli/pkg/commands/setup_manage_local.go

```diff
@@ -54,15 +54,27 @@ func manageLocal() error {
 			defer func() {
 				spinner.Stop(true)
 			}()
-			util.ExecuteCommand(exec.Command(constants.VBOX_MANAGE, "controlvm", constants.VM_NAME, "acpipowerbutton"), "Error stopping VM")
+			err := util.ExecuteCommand(exec.Command(constants.VBOX_MANAGE, "controlvm", constants.VM_NAME, "acpipowerbutton"))
+			if err != nil {
+				fmt.Printf("cellery : %v:\n", err)
```

Comment:
can use `util.ExitWithErrorMessage()` here and other places where we exit the program.

Reply:
Fixed, However, this is temporary. We need to handle errors properly to recover from these errors.